docs/pages/access-controls/guides.mdx

```diff
@@ -9,3 +9,4 @@ description: Detailed guides for configuring Teleport Access Controls
 - [Create certs for CI/CD using impersonation](./guides/impersonation.mdx)
 - [Second Factor - U2F](./guides/u2f.mdx)
 - [Per-session MFA](./guides/per-session-mfa.mdx)
+- [Dual Authorization with ChatOps](./guides/dual-authz.mdx)
```

Comment:
Did you mean Mattermost, or wanted to keep it more generic ChatOps and potentially add other chat software to the guide later?

Reply:
the latter, yes